explorer/k8s/helm/archive-squid/misc/acme-certificate.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-production
+  namespace: explorer-network
+spec:
+  acme:
+    # You must replace this email address with your own.
+    # Let's Encrypt will use this to contact you about expiring
+    # certificates, and issues related to your account.
+    email: alerts@subspace.network
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      # Secret resource that will be used to store the account's private key.
+      name: explorer-issuer-account-key
+    # Add a single challenge solver, HTTP01 using nginx
+    solvers:
+    - http01:
+        ingress:
+          class: nginx
+    # solvers:
```

Comment:
why keep them if you dont use them ?

Reply:
I've removed cluster issuer, but want to keep the DNS resolver in case i use that method